src/test/java/org/openstreetmap/atlas/geography/converters/MultiplePolyLineToPolygonsConverterCommandTest.java

```diff
@@ -0,0 +1,28 @@
+package org.openstreetmap.atlas.geography.converters;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openstreetmap.atlas.streaming.resource.Resource;
+import org.openstreetmap.atlas.streaming.resource.StringResource;
+import org.openstreetmap.atlas.streaming.resource.WritableResource;
+
+/**
+ * @author matthieun
+ */
+public class MultiplePolyLineToPolygonsConverterCommandTest
```

Comment:
Blazing the trail! We have no excuses for other commands now....

Reply:
No excuses 🤪